<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>790</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>790</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>791</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Список учеников</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>781</width>
           <height>471</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="add_student_button">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="search_student_button">
              <property name="text">
               <string>Найти</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Нарезка ключей</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>125</y>
           <width>771</width>
           <height>341</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string>Здесь был процесс по имени Петя</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>781</width>
           <height>101</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Ключи</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="select_file_button">
              <property name="text">
               <string>Выбрать файл</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="select_path_button">
              <property name="text">
               <string>Выбрать путь сохранения</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
            <item>
             <widget class="QPushButton" name="start_cut_button">
              <property name="text">
               <string>Нарезать</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="create_email_button">
              <property name="text">
               <string>Создать рассылку</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Список</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="select_path_button_2">
              <property name="text">
               <string>Выбрать путь сохранения</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_4"/>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox">
              <item>
               <property name="text">
                <string>Все</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Только ФИО</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Телефон</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>E-mail</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="create_list_button">
              <property name="text">
               <string>Вывести</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Хранилище документов</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>781</width>
           <height>471</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="add_document_button">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_3"/>
            </item>
            <item>
             <widget class="QPushButton" name="search_button_2">
              <property name="text">
               <string>Найти</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget_1"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
